presto-main/src/test/java/com/facebook/presto/sql/planner/TestPredicatePushdown.java

```diff
@@ -100,12 +100,8 @@ public void testWithPartitionedTable()
 
         assertPlan(
                 "SELECT orderstatus FROM orders WHERE orderstatus = 'no_such_partition_value'",
-                // TODO this could be optimized to VALUES with so partition matches
                 output(
-                        exchange(
-                                filter(
-                                        "cast(OS AS varchar(23)) = 'no_such_partition_value'",
-                                        tableScan("orders").withAlias("OS", columnReference("orders", "orderstatus"))))),
+                        values("")),
```

Comment:
What `""` stands for?

Reply:
symbols which I would care later in assertion

Reply:
I don't understand. `""` looks like introducing no symbols at all

Reply:
It is a list of aliased which you could then reuse it later. This `""` was a single alias called as empty string.

Fixed that with fixup commit.